pymets/metsdoc.py

```diff
@@ -11,6 +11,7 @@ def __init__(self, value):
     def __str__(self):
         return "%s" % (self.value,)
 
+
```

Comment:
Down below a ways, the docstring for `metsxml2py` should be updated to reflect `BytesIO` vs. `StringIO`.

Reply:
adjusted